### PR TITLE
chore: sync to ab96ea19ca0ea6fc158acdf16a694f6eef181cf7

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Comment by kleyow** [comment]
This has been rebased off of upstream/master instead since rebasing off of v11.3.3 was failing

**Comment by kleyow** [comment]
@eoln

File: LICENSE.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # LICENSE
 
-Copyright © 2017 Bill & Melinda Gates Foundation
+Copyright © 2020 Mojaloop Foundation
```
Comment (eoln) [inline LICENSE.md:3]: I would drop this change if it isn't coming from the `master`

File: src/handlers/positions/handler.js [hunk 2]
```diff
@@ -115,17 +115,27 @@ const positions = async (error, messages) => {
...
-        : (action === Enum.Events.Event.Action.COMMIT ? Enum.Events.ActionLetter.commit
-          : (action === Enum.Events.Event.Action.REJECT ? Enum.Events.ActionLetter.reject
-            : (action === Enum.Events.Event.Action.ABORT ? Enum.Events.ActionLetter.abort
-              : (action === Enum.Events.Event.Action.TIMEOUT_RESERVED ? Enum.Events.ActionLetter.timeout
-                : (action === Enum.Events.Event.Action.BULK_PREPARE ? Enum.Events.ActionLetter.bulkPrepare
-                  : (action === Enum.Events.Event.Action.BULK_COMMIT ? Enum.Events.ActionLetter.bulkCommit
-                    : (action === Enum.Events.Event.Action.BULK_TIMEOUT_RESERVED ? Enum.Events.ActionLetter.bulkTimeoutReserved
-                      : (action === Enum.Events.Event.Action.BULK_ABORT ? Enum.Events.ActionLetter.bulkAbort
-                        : Enum.Events.ActionLetter.unknown)))))))))
+    const actionLetter = action === Enum.Events.Event.Action.PREPARE
```
Comment (eoln) [inline src/handlers/positions/handler.js:118]: this is a masterpiece test - you pass if you exactly know what is going on :-D

**Review by eoln — APPROVED** [review]

File: LICENSE.md (hunk 1 above)
Comment (kleyow) [reply LICENSE.md:3]: It is. Lewis seems to updated the licenses accross many of the repos.

**Pull request merged**